### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by mzahana** [mzahana/mavros_trajectory_tracking#1]
@Jaeyoung-Lim This package integrates mavros_controllers with mav_trajectory_generation. Your testing and comments are appreciated.

**Comment by Jaeyoung-Lim** [comment]
This is awesome! @mzahana were you able to test fly this yet?

From a initial thought, it would be quite cool if we can extract the mission from QGC and then plan the trajectories between the waypoints

**Comment by mzahana** [comment]
Yes. You can also test in simulation using `px4_trajectory_tracking.launch`

**Comment by Jaeyoung-Lim** [comment]
@mzahana I meant by test flying on a real vehicle

**Comment by mzahana** [comment]
Yes. Not exactly the same setup, but similar. This should  work too.

**Comment by mzahana** [comment]
For the QGC feature a PR will be appreciated ;)

**Comment by Jaeyoung-Lim** [comment]
Right, This looks awesome, I will give it a shot
